File: CPP/RPS.cpp
#include <iostream>
#include <tuple> 
using namespace std;

int main(){
	string x,y;
	cin >>x;
	cin >>y;
	
if (x=="rock"){
    if (y=="scissors"){
        cout<<"Player 1 wins"<< endl;
    }
    if (y=="paper"){
        cout<<"Player 2 wins" <<endl;
		}
    if (y=="rock"){
        cout<<"Tie"<< endl;
		}
	}

if (x=="paper"){
        if (y=="scissors"){
            cout<<"Player 2 wins" <<endl;
        }
        if (y=="rock"){
             cout<<"Player 1 wins"<< endl;
         }
        if (y=="paper"){
           cout<<"Tie"<< endl;
       }
   }

if (x=="scissors"){
        if (y=="paper"){
             cout<<"Player 1 wins"<< endl;
         }
        if (y=="rock"){
            cout<<"Player 2 wins" <<endl;
        }
        if (y=="scissors"){
            cout<<"Tie"<< endl;
        }
	}
	return 0;
}

File: PY/RPS.py
r='rock'
p='paper'
s='scissors'

 x=str(input())
y=str(input())
if x==r:
    if y==s:
        print('Player 1 wins')
    elif y==p:
        print('Player 2 wins')
    elif y==r:
        print('Tie')

elif x==p:
        if y==s:
            print('Player 2 wins')
        elif y==r:
            print('Player 1 wins')
        elif y==p:
            print('Tie')
elif x==s:
        if y==p:
            print('Player 1 wins')
        elif y==r:
            print('Player 2 wins')
        elif y==s:
            print('Tie')
